bootstrap-demo.sh

```diff
@@ -43,56 +43,56 @@ if [[ $OSTYPE == "linux-gnu" && $CLOUD_SHELL == true ]]; then
         ## Get user defined Profile
 
         # Kops on GCE?
-        read -e -p "Create GKE Cluster? (Y/N) [y]:" gke  
+        read -e -p "Create GKE Cluster? (Y/N) [y]:" gke
         export GKE_CLUSTER=${gke:-"y"}
 
         # Kops on GCE?
-        read -e -p "Kops on GCE? (Y/N) [${KOPS_GCE:-$KOPS_GCE}]:" kopsg 
+        read -e -p "Kops on GCE? (Y/N) [${KOPS_GCE:-$KOPS_GCE}]:" kopsg
         export KOPS_GCE=${kopsg:-"$KOPS_GCE"}
 
         shopt -s nocasematch
         if [[ ${KOPS_GCE} == y ]]; then
             # GCE Context name
-            read -e -p 'GCE_CONTEXT [remote]:' key
-            export GCE_CONTEXT=${key:-"remote"} 
+            read -e -p 'GCE_CONTEXT [onprem]:' key
+            export GCE_CONTEXT=${key:-"onprem"}
         fi
 
         # Kops on AWS?
-        read -e -p "Kops on AWS? (Y/N) [${KOPS_AWS:-$KOPS_AWS}]:" kopsa 
+        read -e -p "Kops on AWS? (Y/N) [${KOPS_AWS:-$KOPS_AWS}]:" kopsa
         export KOPS_AWS=${kopsa:-"$KOPS_AWS"}
         shopt -s nocasematch
-        
+
         if [[ ${KOPS_AWS} == y ]]; then
 
         # AWS Context name
         read -e -p 'AWS_CONTEXT [external]:' key
-        export AWS_CONTEXT=${key:-"external"} 
+        export AWS_CONTEXT=${key:-"external"}
 
         # AWS Uniquee Bucket Postfix
-        export AWS_RND=${AWS_RND:-"1"}    
+        export AWS_RND=${AWS_RND:-"1"}
         fi
-        
 
-        # Config repo source 
-    
-        read -e -p "Config Repo Source [https://github.com/cgrant/policy-repo]:" reposource 
+
+        # Config repo source
+
+        read -e -p "Config Repo Source [https://github.com/askmeegs/hipster]:" reposource
```

Comment:
TODO - change this to cgrant/policy repo once murog@'s PR lands 

https://github.com/cgrant/hipster